vpr/src/util/vpr_utils.cpp

```diff
@@ -587,10 +587,37 @@ std::tuple<ClusterNetId, int, int> find_pb_route_clb_input_net_pin(ClusterBlockI
     //Walk back from the sink to the CLB input pin
     int curr_pb_pin_id = sink_pb_pin_id;
     int next_pb_pin_id = pb_routes[curr_pb_pin_id].driver_pb_pin_id;
+
+    /* TODO: REMOVE AFTER DEBUGGING IS FINISHED
+     * VTR_LOG("Backtracing clb '%lu' pb_route for net '%s' at pin '%s'...\n",
+     *         size_t(clb),
+     *         g_vpr_ctx.atom().nlist.net_name(pb_routes[sink_pb_pin_id].atom_net_id).c_str(),
+     *         pb_routes[curr_pb_pin_id].pb_graph_pin->to_string().c_str());
+     */
+
     while (next_pb_pin_id >= 0) {
         //Advance back towards the input
         curr_pb_pin_id = next_pb_pin_id;
 
+        /* TODO: REMOVE AFTER DEBUGGING IS FINISHED
```

Comment:
There should be an issue assigned to removing this, unless this is a WIP pull request.